pkg/manager/member/tidb_member_manager.go

```diff
@@ -792,11 +792,11 @@ func (tmm *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, se
 	if err != nil {
 		return err
 	}
-	if upgrading && tc.Status.TiKV.Phase != v1alpha1.UpgradePhase &&
+	if tc.TiDBStsDesiredReplicas() != *set.Spec.Replicas {
```

Comment:
`tidb_upgrader.go` needs to update to skip upgrading if scaling is in progress.

Reply:
Could you please also help extend the stability [case](https://github.com/pingcap/tidb-operator/blob/master/tests/e2e/tidbcluster/stability.go#L593)?